kart/point_cloud/metadata_util.py

```diff
@@ -307,3 +320,22 @@ def extract_format(tile_format):
         if "format" in tile_format:
             return tile_format["format"]
     return tile_format
+
+
+def find_extra_bytes_vlr(metadata):
+    return find_vlr(metadata, "LASF_Spec", 4)
+
+
+def find_vlr(metadata, user_id, record_id):
+    for key, value in metadata.items():
+        if not key.startswith("vlr"):
+            continue
+        if value["user_id"] == user_id and value["record_id"] == record_id:
+            return base64.b64decode(value["data"])
+
+
+def _remove_nones(dict_to_modify):
```

Comment:
I'm generally against functions that modify arg in-place _and_ return it - do one or the other to avoid causing confusion

This function's a natural fit for returning a dict comprehension.

Reply:
Done